### DISCUSSION TITLE
How to serialize/deserialize std::map<std::string, std::any>

### DISCUSSION THREAD

**Discussion by adcortez** [nlohmann/json#3801] [Q&A]
Hello I am having an issue when I try to convert my JSON into a C++ class

For example:

I receive a JSON like this.      ->   "allValues":{"value1":123,"value2":"true","value3":4.5,"value4":false}

and then in my class I have a property declared like this -> std::map<std::string, std::any> allValues;

But I got errors:
1: No matching member function for call to 'get' in the json.hpp, function: from_json_tuple_impl
2: No viable overloaded '='

I am using OR_EXTEND_DEFINE_TYPE_INTRUSIVE_WITH_DEFAULT

Thanks for your help

**Comment by falbrechtskirchinger** [comment]
There's no built-in conversion for `std::any` and implicit conversion to `std::any` was explicitly disabled some time ago. You should still be able to define your own conversion functions for `std::any`.
